homeassistant/components/dyson/sensor.py

```diff
@@ -125,19 +133,11 @@ def unique_id(self):
         return f"{self._device.serial}-{self._sensor_type}"
 
     @property
-    def unit_of_measurement(self):
```

Comment:
We shouldn't remove these properties. They are part of the entity API that we need to use.